src/Equinox.Core/Tracing.fs

```diff
@@ -0,0 +1,30 @@
+module Equinox.Core.Tracing
+
+open System.Diagnostics
+
+let source = new ActivitySource("Equinox")
+
+[<System.Runtime.CompilerServices.Extension>]
+type ActivityExtensions =
+
+    [<System.Runtime.CompilerServices.Extension>]
+    static member AddAttempt(act: Activity, attempt: int) = act.AddTag("eqx.op_attempt", attempt)
```

Comment:
this is _not_ the `op_` attempt - maybe entitle it transact_attempt?

Reply:
We could put it down as "eqx.attempt" as the retries will be on a separate span

Reply:
yeah, that works, as long as one is mentally tagged as different